### DISCUSSION TITLE
Num — Missing Laravel Helper

### DISCUSSION THREAD

**Discussion by 514sid** [HassanZahirnia/laravel-package-ocean#116] [Package Suggestions]
The original idea was to extend Laravel helpers like `Str`, but the pull request was rejected with the comment "This might be a good candidate for a package 👍" (https://github.com/laravel/framework/pull/47997).

I've improved this helper, added tests that cover most cases I found across the web and published it as a package.

https://github.com/514sid/num

The package is ready for use in a production environment.

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
